README.md

```diff
@@ -1,34 +1,73 @@
-# bookay
 
-[![Build Status](https://github.com/jaynetics/bookay/workflows/tests/badge.svg)](https://github.com/jaynetics/bookay/actions)
+<br>
 
-bookay is a set of free, open-source tools for self-hosting your bookmarks.
+<div align = center>
 
-[
-  ![Screenshot thumbnail showing the webapp and browser plugin](
-    https://user-images.githubusercontent.com/10758879/113178657-1528d000-924f-11eb-9a02-2c0e4e504074.png
-  )
-](https://user-images.githubusercontent.com/10758879/113178042-77350580-924e-11eb-820f-298da2a2631d.png)
+[![Badge Build]][Actions]   [![Badge Heroku]][Heroku]
 
-## Why?
+<br>
 
-- a bookmarking service is useful if you use varying browsers or devices
-- there is currently no free service that supports nested folders
-- self-hosting your bookmarks offers the best privacy
+# Bookay
+
+***Self-Host*** *your* ***Bookmarks***
+
+<br>
+<br>
+
+
+<img
+    src = 'https://user-images.githubusercontent.com/10758879/113178042-77350580-924e-11eb-820f-298da2a2631d.png'
+    width = 500
+/>
+
+</div>
+
+<br>
+
+## Why
+
+- A bookmarking service is useful if you use varying browsers or devices
+
+- There is currently no free service that supports nested folders
+
+- Self - hosting your bookmarks offers the best privacy
+
+<br>
+<br>
 
 ## Features
 
-- easily deployed [backend](./server/)
-- customizable, keyboard-friendly [browser plugin](./plugin/)
-- lightweight (~50 kB) [web app](./webapp/)
-- support for nested folders
-- data import and export
-- scan for dupes and broken links
-- support for multiple users per instance
-- responsiveness and dark mode
+- Customizable, keyboard-friendly **[Browser Plugin]**
```

Comment:
the spacing is another thing that makes the README a tiny bit less "maintainable" - when changing such spaced lists, people have to keep the spacing right (a small effort, but in a larger project than this one it could be problematic)

Reply:
> the spacing is another thing that makes the README a tiny bit less "maintainable"

- Well you can just revert back to the good old README.txt and forget 
  about formatting, that would be the easiest to maintain option.

- When changing it you could also just not adjust it like it,
  then there wouldn't be a difference to how it was before.

> a small effort, but in a larger project than this one it could be problematic

This indicates to me the association of big project = big README,
which is usually a type of downfall many projects have and is in
fact not necessary in the slightest.

Btw, what did you expect me to change 
to begin with, unorder them again?